app/vmalert/utils.go

```diff
@@ -103,9 +103,40 @@ func requestToCurl(req *http.Request) string {
 
 	for _, k := range keys {
 		cw.add("-H")
+		if isSecreteHeader(k) {
+			req.Header[k] = hideSecretes(req.Header[k])
```

Comment:
Is it safe to modify original request? Why do not use copy of header value ?

Reply:
logic was updated